<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimaryFaceUI</class>
 <widget class="QWidget" name="PrimaryFaceUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
     <property name="spacing">
      <number>50</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="topMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <property name="bottomMargin">
      <number>50</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
     border-style: solid ;
     border-width: 2px;
     border-color:rgb(85, 170, 255);  
	 background-color: rgb(85, 170, 255);
	 font: 15pt &quot;华文行楷&quot;;
	 border-bottom-width:0px;
 	 border-top-left-radius:15px;
	 border-top-right-radius:15px;
	 color: rgb(255, 255, 255);
	 text-align:center;
}</string>
         </property>
         <property name="text">
          <string>公告</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="informTextEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QTextEdit{
     border-style: solid ;
     border-width: 2px;
     border-color:rgb(85, 170, 255);
     background-color:rgba(255, 170, 255,00);
	 font: 13pt &quot;华文行楷&quot;;
	 border-bottom-left-radius:15px;
	 border-bottom-right-radius:15px;
	 padding:10px;
	
	color: rgb(0, 170, 255);
}</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;title&gt;mystery - 爱N就在天边的博客&lt;/title&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'华文行楷'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;测试看一看你会不会换行&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
     border-style: solid ;
     border-width: 2px;
     border-color:rgb(85, 170, 255);  
	 background-color: rgb(85, 170, 255);
	 font: 15pt &quot;华文行楷&quot;;
	 border-bottom-width:0px;
 	 border-top-left-radius:15px;
	 border-top-right-radius:15px;
	 color: rgb(255, 255, 255);
	 text-align:center;
}</string>
         </property>
         <property name="text">
          <string>版本更新</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="updataTextEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QTextEdit{
     border-style: solid ;
     border-width: 2px;
     border-color:rgb(85, 170, 255);
     background-color:rgba(255, 170, 255,00);
	 font: 13pt &quot;华文行楷&quot;;
	 border-bottom-left-radius:15px;
	 border-bottom-right-radius:15px;
	 padding:10px;
	 
	
	color: rgb(85, 170, 255);
}</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;title&gt;mystery - 爱N就在天边的博客&lt;/title&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'华文行楷'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;测试看一看你会不会换行&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="insertButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">.QPushButton{
	background-color: rgb(85, 170, 255);
	border-color: rgb(85, 170, 255);
	border-width:2px;
	border-style:solid;
	border-radius:5px;
	padding:5px;
	font: 75 9pt &quot;微软雅黑&quot;;
}
.QPushButton:hover{
	
	border-color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>帮助与反馈</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="insertButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">.QPushButton{
	background-color: rgb(85, 170, 255);
	border-color: rgb(85, 170, 255);
	border-width:2px;
	border-style:solid;
	border-radius:5px;
	padding:5px;
	font: 75 9pt &quot;微软雅黑&quot;;
}
.QPushButton:hover{
	
	border-color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>下载新版本</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
